modules/test/playwright/tests/frontend-js-item-selector-web/main/itemSelectorModal.spec.ts

```diff
@@ -100,24 +109,32 @@ test('Item Selector Modal with single selection', async ({
 	await test.step('Check that a single item can be selected in the Cards visualization mode', async () => {
 		await expect(itemSelectorSamplePage.modal.selectButton).toBeDisabled();
 
-		await itemSelectorSamplePage.page
-			.locator('.custom-radio')
-			.first()
-			.click();
+		const items = itemSelectorSamplePage.page.locator(
+			'.card:has(>.custom-radio)'
+		);
+
+		const numOfItems = await items.count();
+
+		expect(numOfItems > 1).toEqual(true);
+
+		const testItem = items.getByText('Test', {exact: true});
+
+		const itemUser = items.getByText(user?.givenName, {exact: true});
```

Comment:
Can we update these names so they are more consistent and easier to understand. Maybe `user1Radio` `user2Radio` or `user1Card` `user2Card`?